katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -42,6 +42,10 @@
             "type": "integer",
             "minimum": 1
         },
+        "nonneg_number": {
+            "type": "number",
+            "minimum": 0.0
```

Comment:
I wonder if we should actually allow 0 - could we get a target in the list that is never actually observed ? Some sort of sensible minimum like 60second could make sense, although I am sure we could imagine a pathological case in which this wouldn't work.

Reply:
Good point - strictly positive makes more sense. Not sure why I thought it would be a good idea to allow 0, as it'll probably cause things to crash. I've changed it, although I've kept the nonneg_number definition since I put it to use elsewhere.